internal/check/must.go

```diff
@@ -20,7 +20,19 @@ func MustAnyValue[T any](v any, err error) T {
 	return v.(T)
 }
 
-func MustOK[T any](v any, ok bool) T {
+func MustOK(ok bool) {
```

Comment:
Changing this function shouldn't be necessary. Could you revert the change? In tests we can use `require.True` instead.
I don't think it's used anywhere. Let's remove it when we do the cleanup.